packages/async-operations/tests/test-async_operations.js

```diff
@@ -152,18 +141,13 @@ test.serial('startAsyncOperation uploads the payload to S3', async (t) => {
     dynamoTableName: dynamoTableName,
```

Comment:
Still seeing this all over the place in this file.

Reply:
f6c38ac936